tasks/read-client-files.yml

```diff
@@ -5,12 +5,21 @@
   register: openvpn_read_ca_file_results
   changed_when: false
 
+- name: Read TLS-auth key
+  slurp:
+    src: "{{ openvpn_keydir }}/{{ openvpn_tls_key }}"
+  no_log: true
+  register: openvpn_read_tlsauth_file_results
+  changed_when: false
+  when: openvpn_tls_auth
+
 - name: Set CA file contents as fact.
   set_fact:
     openvpn_ca_file_contents: "{{ openvpn_read_ca_file_results.stdout }}"
+    openvpn_tls_auth_file_contents: "{{ openvpn_read_tlsauth_file_results['content'] | b64decode | default('') }}"
 
 - name: Read client cert files.
-  command: cat {{ openvpn_etcdir }}/keys/{{ item }}.crt
+  command: sed -n '/-----BEGIN CERTIFICATE-----/,/-----END CERTIFICATE-----/p' {{ openvpn_keydir }}/{{ item }}.crt
```

Comment:
Sorry I wasn't clear with what the change was doing... the `crt` file that is getting read in has a lot of extra text. Similar to the output when you run `openssl -in cert -text`, looking something like this:
```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 6 (0x6)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: ....
        Validity
            Not Before: ....
            Not After : ....
        Subject: ....
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (2048 bit)
                Modulus:
-----BEGIN CERTIFICATE-----
[...]
-----END CERTIFICATE-----
```

my change just reads in everything around the `BEGIN CERTIFICATE` / `END CERTIFICATE` blocks... including those headers